director/discovery.go

```diff
@@ -27,25 +27,55 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-type DiscoveryResponse struct {
+type OpenIdDiscoveryResponse struct {
 	Issuer  string `json:"issuer"`
 	JwksUri string `json:"jwks_uri"`
 }
 
 const (
-	directorDiscoveryPath string = "/.well-known/openid-configuration"
-	directorJWKSPath      string = "/.well-known/issuer.jwks"
+	openIdDiscoveryPath     string = "/.well-known/openid-configuration"
+	federationDiscoveryPath string = "/.well-known/pelican-configuration"
+	directorJWKSPath        string = "/.well-known/issuer.jwks"
 )
 
-// Returns links related to director's authentication, including the link
-// to get the public key from the director
-func discoveryHandler(ctx *gin.Context) {
+func federationDiscoveryHandler(ctx *gin.Context) {
 	directorUrl := param.Federation_DirectorUrl.GetString()
```

Comment:
Should this fall back to the external web URL if this is the director (probably more of a question in the config.go, not actually putting the logic here?).